cmd/scale.go

```diff
@@ -34,8 +33,8 @@ type scaleCmd struct {
 
 	// user input
 	resourceGroupName    string
-	deploymentDirectory  string
 	newDesiredAgentCount int
+	deploymentDirectory  string
```

Comment:
apiModelPath is a user input now so it should move up to this section for correctness

Reply:
done